### PR TITLE
Move rpc/Context.h out of impl/

### PR DESCRIPTION
**Pull request by rec**
Reviewers: @vinniefalco @scottschurr

### CODE REVIEW

**Comment by vinniefalco** [comment]
`scons vcxproj`

File: src/ripple/rpc/impl/RPCHandler.cpp [hunk 1]
```diff
@@ -33,7 +32,6 @@
 #include <ripple/net/InfoSub.h>
 #include <ripple/net/RPCErr.h>
 #include <ripple/protocol/JsonFields.h>
-#include <ripple/rpc/impl/Context.h>
```
Comment (scottschurr) [inline src/ripple/rpc/impl/RPCHandler.cpp]: Nice catch on the extra includes.

**Comment by scottschurr** [comment]
:+1:
(Assuming you run `scons vcxproj` like Vinnie said.)

**Comment by rec** [comment]
vcxprojected and sconsed.

**Comment by vinniefalco** [comment]
Travis error is spurious

**Comment by lukapusic** [comment]
I'm not sure if this has anything to do with it, but I can't get 0.27.0-b11 to compile on Ubuntu 14.04. 0.27.0-b5 compiled without problems.

```
g++: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-4.8/README.Bugs> for instructions.
scons: *** [build/gcc.release/src/ripple/unity/beast.o] Error 4
scons: building terminated because of errors.
```

**Comment by rec** [comment]
Pushed a change with the new name, probably will convince Travis.

**Comment by vinniefalco** [comment]
Oops... the commit message is wrong now.
Suggested: "Move Context.h to rpc/"
Rationale: The previous message has the original location wrong. Better to omit the original location entirely, since it can be determined from looking at the changes. This way we avoid baking stale information into the first line of commit messages.

**Comment by rec** [comment]
Good catch!  I changed it to Move Context.h up into rpc/, which has a tiny bit more information even.

**Comment by nbougalis** [comment]
Merged as 06207da18566e487c857aff739ede26ebf772a76

**Pull request closed without merging**